packages/react-components/src/components/DatePicker/DatePicker.spec.tsx

```diff
@@ -0,0 +1,39 @@
+import { within } from '@testing-library/react';
+import * as React from 'react';
+import { render, userEvent } from 'test-utils';
+import { vi } from 'vitest';
+import { DatePicker } from './DatePicker';
+import { IDatePickerProps } from './types';
+
+describe('<DatePicker> component', () => {
```

Comment:
Can we think of a more advanced set of tests? Alternatively, we could cover it in a separate PR, however, first, we need to make sure manual tests are passed.

Reply:
will create separate task for that